jitbuilder/compile/Method.hpp

```diff
@@ -166,7 +167,7 @@ class ResolvedMethod : public ResolvedMethodBase, public Method
                                TR::SymbolReferenceTable *symRefTab);
 
    protected:
-   const char *_fileName;
+   std::string _fileName;
```

Comment:
We could change some of that story by creating a JitBuilder bytecode set or persistent structure (which you've proposed before for other reasons) and then having the JitBuilder il generator become a visitor of that structure that generates OMR compiler IL. That's more analogous to how LLVM IR builders work.

I think we should enable this path (by implementing a particular kind of "standard" MethodBuilder that parses said structure to generate IL), but I don't (yet anyway) think we should change JitBuilder to operate in only that way.